docs/token-format.md

```diff
@@ -0,0 +1,48 @@
+# Token Format
+
+Clients can expect to exchange JWTs from one security domain for OAuth 2.0 access tokens issued by identity-manager-sts. This document outlines the parameters for token requests, how they affect the response and the meaning of the claims contained in the issued access token.
+
+We anticipate that these tokens will be used to inform further authorization decisions later in the API gateway stack. While RFC 8693 allows for issuing ID tokens, access tokens give us the ability to use standard claims to provide information necessary for making authorization decisions.
+
+### Token Request
+
+This is as defined in [RFC 8693][rfc-8693]. At minimum, a token request includes the following parameters:
+
+- `grant_type` (always set to `urn:ietf:params:oauth:grant-type:token-exchange`
+- `subject_token`
+- `subject_token_type`
+
+Issued access tokens can be restricted further through the use of `audience`, `resource`, and `scope` parameters in the token request. `audience` and `resource` restrict the class of resources for which the issued token may be used. `scope` restricts the type of actions the token can perform on resources.
+
+For the first iteration of identity-manager-sts, we are only supporting the use of `resource`, which will allow us to expose a set of services that a user may use to restrict the context in which the token is valid. The `resource` parameter maps to the `aud` claim on the issued token, as outlined in [RFC 9068][rfc-9068].
+
+
+### Token Response
+
+The token returned is an [RFC 9068][rfc-9068]-compliant access token JWT with the following claims:
+
+| claim              | description                                                                                         |
+|--------------------|-----------------------------------------------------------------------------------------------------|
+| iss                | FQDN for identity-manager-sts issuer                                                                |
+| iat                | The time the token was issued                                                                       |
+| jti                | Unique ID for the token issued                                                                      |
+| exp                | Token expiration time                                                                               |
+| sub                | Subject of the token in the infratographer namespace<br>example "sub": `infratographer:user:{uuid}` |
+| aud                | Resources on which that token may operate                                                           |
+| client_id          | ID of the client requesting the token                                                               |
+| infratographer.sub | Private claim which indicates the subject be used in policy enforcement                             |
```

Comment:
You probably want to add a link to the subsection bellow.

Reply:
After thinking about this some more, I'm less convinced that `infratographer.sub` is the right abstraction here. Existing standards already provide claims for control over groups, roles, and individual resources: those seem like they would give us more than enough flexibility to define robust access management policies.